### ISSUE TITLE
Update to 1.18

### ISSUE THREAD

**Issue by Anoaxx** [Siphalor/amecs#48]

**Comment by Klotzi111** [comment]
My forked version is compatible with Minecraft 1.18.
See https://github.com/Siphalor/amecs/issues/47#issuecomment-989053383

**Comment by Anoaxx** [comment]
> My forked version is compatible with Minecraft 1.18. See [#47 (comment)](https://github.com/Siphalor/amecs/issues/47#issuecomment-989053383)

There is a problem... I tested this so much... yours Amecs and Amecs Api causes minecraft to crash. Nothing more. Deleting every mod(expect fabric) changes nothing. deleting forked amecs solves problem.

**Comment by Klotzi111** [comment]
Then please tell me the Minecraft and Fabric version you are using. Because for me it works.
And send the game log file. The crash report is required to find the problem

**Comment by Anoaxx** [comment]
minecraft 1.18 + fabric loader 0.12.9(i found that this is not latest version like about 5 mins ago) + fabric api 0.44.0

**Comment by Anoaxx** [comment]
+ logs from folder logs or something else?

**Comment by Klotzi111** [comment]
In the `logs` folder there should be a `latest.log` file. Send that

**Comment by Anoaxx** [comment]
[latest.log](https://github.com/Siphalor/amecs/files/7697601/latest.log)

**Comment by Anoaxx** [comment]
i update fabric loader but still nothing changes

**Comment by Klotzi111** [comment]
Oh yea I see the Problem. I made a mistake exporting the mod. I am working on it...

**Comment by Anoaxx** [comment]
You're welcome! <3

**Comment by Klotzi111** [comment]
I (hopefully) fixed it. You can download now again

**Comment by Anoaxx** [comment]
im on it

**Comment by Anoaxx** [comment]
from releases on github?

**Comment by Klotzi111** [comment]
yes. As you did before

**Comment by Anoaxx** [comment]
okey, it works on 1.18, but i didn't test this on 1.18.1

**Comment by Klotzi111** [comment]
It "should" work there too

**Comment by Klotzi111** [comment]
I am glad we fixed this.
Thanks for reporting this problem. Others might have had the problem too but did not report it.

**Comment by Anoaxx** [comment]
No problem <3

**Comment by mtb-xt** [comment]
@Klotzi111 I'm sorry, I'm trying to use your fork too, but it crashes, I think it conflicts with 'interactic' somehow?

```
[05/01/2022 16:07:51 PM] [main/WARN] @Redirect conflict. Skipping interactic.mixins.json:MinecraftClientMixin->@Redirect::handleDropPower(Lnet/minecraft/class_746;Z)Z from mod interactic with priority 1000, already redirected by amecs.mixins.json:MixinMinecraftClient->@Redirect::dropSelectedItem(Lnet/minecraft/class_746;Z)Z from mod amecs with priority 1000
[05/01/2022 16:07:51 PM] 	at Not Enough Crashes deobfuscated stack trace.(1.17.1+build.39)0
[05/01/2022 16:07:51 PM] Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Redirector handleDropPower(Lnet/minecraft/class_746;Z)Z in interactic.mixins.json:MinecraftClientMixin from mod interactic failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap interactic-refmap.json

```

Full log is here -> https://paste.atlauncher.com/view/a6a50da4-22d2-4db8-bb63-b7e75d0291e5

Also reported at interactic's repo - https://github.com/glisco03/interactic/issues/5

**Comment by Klotzi111** [comment]
You are right. Amces seems to be incompatible with interactic. But the crash in the linked issue on interactic is not the same.

If I find some time I will look into that. But do not expect it too soon

**Comment by TechPro424** [comment]
This mod has a 1.18 version now, I think the issue can be closed


![Screenshot 2022-08-11 111006](https://user-images.githubusercontent.com/86620085/184070788-6436c795-cb02-4295-b0d9-46c611990d14.png)


(Sorry for the bad quality marking on the image)

**Comment by Siphalor** [comment]
Well, yes it can indeed be closed xD
Thanks for the report four years ago ^^
